.storybook/main.ts

```diff
@@ -0,0 +1,27 @@
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-mdx-gfm'],
+  features: {
+    buildStoriesJson: true,
+  },
+  previewHead: (head) => `
+    ${head}
+    <link rel="stylesheet" type="text/css" title="themeCss" href="core-theme.css" />
+  `,
+  stories: [
+    '../packages/design-system/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../packages/ds-healthcare-gov/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+    '../packages/ds-medicare-gov/src/**/*.stories.@(js|jsx|ts|tsx|mdx)',
+  ],
+  staticDirs: ['./static/', '../packages/design-system/src'],
```

Comment:
This actually clobbers a fix I recently made, which was removing the `'../packages/design-system/src'` item